modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua

```diff
@@ -356,6 +356,20 @@ mode:value("ap", translate("Access Point"))
 mode:value("sta", translate("Client"))
 mode:value("adhoc", translate("Ad-Hoc"))
 
+ssid = s:taboption("general", Value, "ssid", translate("<abbr title=\"Extended Service Set Identifier\">ESSID</abbr>"))
```

Comment:
Why are you replicating the SSID option here? It is already declared a few lines up. Please add the extra dependencies to the existing `ssid` option object instead.

Reply:
Hi @jow- ,
Have added requested changes. Kindly review it.

    Kishan Go.